model/ResultsService.php

```diff
@@ -78,6 +80,8 @@ class ResultsService extends OntologyClassService
     public const FILTER_END_FROM = 'endfrom';
     public const FILTER_END_TO = 'endto';
 
+    public const OPTION_ALLOW_SQL_RESULT = 'allow_sql_export';
```

Comment:
let's rename it `OPTION_ALLOW_SQL_EXPORT` ?

Reply:
done